website/docs/d/public_ip.html.markdown

```diff
@@ -107,4 +107,5 @@ output "public_ip_address" {
 * `idle_timeout_in_minutes` - Specifies the timeout for the TCP idle connection.
 * `fqdn` - Fully qualified domain name of the A DNS record associated with the public IP. This is the concatenation of the domainNameLabel and the regionalized DNS zone.
 * `ip_address` - The IP address value that was allocated.
+* `ip_version` - The IP Version to use, IPv6 or IPv4. Must be used with a 'dynamic' allocation type.
```

Comment:
_minor_ since this is only retrieving information about the Public IP - can we update this to a slightly different tense (and remove the information about `dynamic` being required, since this is readonly):

```
* `ip_version` - The IP version being used, for example `IPv4` or `IPv6`
```